cou/utils/juju_utils.py

```diff
@@ -641,6 +198,15 @@ async def _get_unit(self, name: str) -> Unit:
 
         return unit
 
+    async def check_model_name(self) -> None:
+        """Check model name.
+
+        This function should be used if None was used for model name, so it mean that
+        default model will be used.
```

Comment:
```suggestion
        Get default model's name if no model name was explicitly passed in.
```

Reply:
I changed it to `Set ...` since we are setting the `_name` value.